modules/browser/BrowserCapabilities.js

```diff
@@ -47,6 +48,17 @@ export default class BrowserCapabilities extends BrowserDetection {
         return !this.isEdge() && !this.isFirefox();
     }
 
+    /**
+     * Checks if the current browser is "Chrome like", that is, it's either
+     * Chrome / Chromium or uses it as its engine.
+     */
+    isChromeLike() {
```

Comment:
we have both this and https://github.com/jitsi/js-utils/pull/6 ?

Reply:
I guess what I don't understand is if we already detect chrome-like things as chrome, that means this check is effectively the same as:

```
 isChromeLike() {
    return this.isChrome();
}
```
no?  electron/js/opera will already be detected as chrome-like and therefore chrome, no? so what is this adding?  or maybe i'm missing something.